pom.xml

```diff
@@ -20,26 +21,51 @@
   </scm>
   
   <properties>
-    <jenkins.version>2.60.3</jenkins.version>
+    <bom>2.204.x</bom>
+    <jenkins.version>2.204.6</jenkins.version>
```

Comment:
Can we bring this up an LTS that is within the last year? 2.249.x?

Reply:
sure